scripts/docker/omvll-ndk/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:22.04
+LABEL maintainer="weliveindetail <stefan.graenitz@gmail.com>"
+
+RUN apt update && DEBIAN_FRONTEND=noninteractive \
+    apt install -y --no-install-recommends \
+        clang-14 libc++-14-dev libc++abi-14-dev llvm-14-tools \
+        cmake ninja-build git python3-pip zlib1g-dev && \
+    apt clean autoclean && apt autoremove --yes
+
+RUN python3 -m pip install lit && lit --version
+RUN git config --global --add safe.directory /o-mvll
+
+# Copy in missing test dependencies from a local resource
+# FIXME: Once we update the deps-package, these 3 must be included
+RUN mkdir -p /test-deps/bin
+COPY llvm-config /test-deps/bin
+COPY clang /test-deps/bin
+RUN ln -s /test-deps/bin/clang /test-deps/bin/clang++
```

Comment:
I bootstrapped this one from NDKr25c sources:
```
> llvm-toolchain-android/out/stage2-install/bin/clang --version | head -2
Android (dev, based on r450784d1) clang version 14.0.7 (https://android.googlesource.com/toolchain/llvm-project 4c603efb0cca074e9238af8b4106c30add4418f6)
Target: x86_64-unknown-linux-gnu
```
In the future we should probably just use the one from the official release..